grails-app/domain/org/bbop/apollo/FeatureLocation.groovy

```diff
@@ -1,14 +1,15 @@
 package org.bbop.apollo
 
 class FeatureLocation {
-
+    static mapping = {
+        length formula: 'FMAX-FMIN'
```

Comment:
Is there any reason why we can't just make a strong assumption that fmax is greater than fmin?